js/anattadesign/awesomecheckout/opcheckout.js

```diff
@@ -1763,3 +1763,21 @@ jQuery.validator.addClassRules({
 		creditcard2: ''
 	}
 });
+
+// Additions to support MasterCard series 2
+var cc2 = jQuery.validator.methods['creditcard2'];
+jQuery.validator.addMethod('creditcard2', function (value, element, param) {
+
+    var cardName = param;
+
+    if (typeof(cardName) !== 'string' || typeof(cardName) == 'string' && cardName == '') {
+        cardName = ccType(value);
+    }
+
+    if (cardName === 'MasterCard') {
+        return /^(5[1-5][0-9]{14}|2(22[1-9][0-9]{12}|2[3-9][0-9]{13}|[3-6][0-9]{14}|7[0-1][0-9]{13}|720[0-9]{12}))$/.test(value);
```

Comment:
So I don't really know what I can do here except assume your regex is good. Maybe a comment would be a little helpful? I did a quick search and I have seen the regex's written a few different ways.